crypto/Kconfig

```diff
@@ -1038,6 +1038,16 @@ config CRYPTO_SHA3
 	help
 	  SHA-3 secure hash algorithms (FIPS 202, ISO/IEC 10118-3)
 
+config CRYPTO_KECCAK
```

Comment:
We should also revert this after the new tools library changes.

Reply:
I'll do the revert here: https://github.com/cartesi/linux/pull/4